packages/grafana-schema/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "author": "Grafana Labs",
+  "license": "Apache-2.0",
+  "name": "@grafana/schema",
+  "version": "8.1.0-pre",
+  "description": "Grafana Schema Library",
+  "keywords": [
+    "typescript"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "http://github.com/grafana/grafana.git",
+    "directory": "packages/grafana-schema"
+  },
+  "main": "src/index.ts",
+  "types": "src/index.ts",
+  "scripts": {
+    "build": "grafana-toolkit package:build --scope=schema",
+    "bundle": "rollup -c rollup.config.ts",
+    "clean": "rimraf ./dist ./compiled",
+    "docsExtract": "mkdir -p ../../reports/docs && api-extractor run 2>&1 | tee ../../reports/docs/$(basename $(pwd)).log",
+    "typecheck": "tsc --noEmit"
+  },
+  "dependencies": {},
+  "devDependencies": {
```

Comment:
OK, made couple of comments below about different dependencies, but it feels like ATM only dependencies that we need are:
- @grafana/tsconfig
- @grafana/eslint-config (not sure how eslint's gonna work with auto generated files, maybe it's not needed at all)
- typescript